src/Renderer/TrafficRenderer.cpp

```diff
@@ -59,13 +60,14 @@ TrafficRenderer::Draw(Canvas &canvas, const TrafficLook &traffic_look,
       canvas.Select(traffic_look.alarm_brush);
       break;
     case FlarmTraffic::AlarmType::NONE:
-      if (traffic.relative_altitude > (const RoughAltitude)50) {
-        canvas.Select(traffic_look.safe_above_brush);
-      } else if (traffic.relative_altitude > (const RoughAltitude)-50) {
-        canvas.Select(traffic_look.warning_in_altitude_range_brush);
-      } else {
-        canvas.Select(traffic_look.safe_below_brush);
-      }
+        if(colorful_traffic)
```

Comment:
More bad indent and bad coding style. Please be more careful with that. It's everywhere.